src/borg/patterns.py

```diff
@@ -235,7 +235,9 @@ class PathPrefixPattern(PatternBase):
     PREFIX = "pp"
 
     def _prepare(self, pattern):
-        self.pattern = os.path.normpath(pattern).rstrip(os.path.sep) + os.path.sep
+        sep = os.path.sep
+
+        self.pattern = (os.path.normpath(pattern).rstrip(sep) + sep).lstrip(sep)  # sep at beginning is removed
```

Comment:
how about:
- normpath
- lstrip
- rstrip
- add sep

would not require parens like seen above.

Reply:
ah, ok, I overlooked that case.